platform/darwin/src/MGLShapeSource.h

```diff
@@ -1,5 +1,6 @@
 #import "MGLSource.h"
 
+#import "MGLFoundation.h"
```

Comment:
This header is new in the master branch, hence the build failures. In the release-ios-v3.4.0 branch, you don’t need it.